Common.prf

```diff
@@ -18,8 +18,8 @@ CONFIG(release, debug|release) {
   QMAKE_CXXFLAGS += -O3
 }
 
-# Enable C++14
-CONFIG += c++14
+# Enable C++17
```

Comment:
The problem with c++17 is that it wont work on older compilers, typically available in older Linux OS versions.
Also not available in the version of MinGW we are currently shipping on windows.
Either the entire FMIWraper must be optional depending on available C++ version, and the build system must choose automatically. Or the functionality must be rewritten with nasty ifdefs and either posix or MSC runtime calls.

Reply:
We can stick with old C++ code then I guess. I just need to replace the temp directory stuff and some loops and auto-variables.

Reply:
Yes for all components we should assume that it may run on old system.